test/JDBC/expected/pivot-before-15_7-16_3-vu-verify.out

```diff
@@ -0,0 +1,1582 @@
+use pivot_test
+GO
+
+-- 2 column in src table pivot
+SELECT 'OrderNumbers' AS OrderCountbyStore, [1] AS STORE1, [2] AS STORE2, [3] AS STORE3, [4] AS STORE4, [5] AS STORE5
+FROM
+(
+    SELECT StoreID, OrderID
+    FROM StoreReceipt
+)AS SrcTable
+PIVOT (
+    COUNT (OrderID)
+    FOR StoreID IN ([1], [2], [3],[4], [5])
+) AS pvt
+GO
+~~START~~
+varchar#!#int#!#int#!#int#!#int#!#int
+OrderNumbers#!#19#!#19#!#19#!#16#!#14
+~~END~~
+
+
+-- 3 column in src table pivot
+SELECT EmployeeID, [2] AS STORE2, [3] AS STORE3, [4] AS STORE4, [5] AS STORE5, [6] AS STORE6
```

Comment:
Please add ORDER BY for most of the test cases

Reply:
Added ORDER BY to all test cases